src/shared/components/deletion-confirmation/DeletionConfirmation.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+
+import { PrimaryButton } from 'src/shared/components/button/Button';
+
+import styles from './DeletionConfirmation.scss';
+
+export type DeletionConfirmationProps = {
+  warningText?: string;
+  confirmationText?: string;
+  cancellationText?: string;
```

Comment:
Minor, but do you to change it to `confirmText`/`cancelText`?

Reply:
i prefer `confirmText` / `cancelText` but lets leave it as it is since it is another variable debate which wouldn't make much difference here.

Reply:
I preferred confirmText and cancelText as well, but convinced myself to call them confirmation and cancellation, as nouns. Will be happy if you change them to confirm and cancel texts.

Reply:
✔